tests/Unit/Helpers/Domain/DomainTestHelpers.cpp

```diff
@@ -33,6 +33,7 @@
 #include "Domain/Structure/Neighbors.hpp"
```

Comment:
I'm worried that for domains people will call these functions but no tests will happen. They'll then deduce "All tests pass" and move on. Can we add `ERROR`s to guard against that?

Reply:
no; this just skips a subset of the tests that use corner iterators which make no sense for domains with no corners.  I will rewrite these test functions not to use corner iterators before I put in additional domains, but not now...